src/sentry/search/events/constants.py

```diff
@@ -209,10 +209,10 @@ class ThresholdDict(TypedDict):
     "measurements.stall_stall_total_time": "d:transactions/measurements.stall_total_time@millisecond",
     "measurements.ttfb": "d:transactions/measurements.ttfb@millisecond",
     "measurements.ttfb.requesttime": "d:transactions/measurements.ttfb.requesttime@millisecond",
-    "spans.browser": "d:transactions/breakdowns.span_ops.browser@millisecond",
-    "spans.db": "d:transactions/breakdowns.span_ops.db@millisecond",
-    "spans.http": "d:transactions/breakdowns.span_ops.http@millisecond",
-    "spans.resource": "d:transactions/breakdowns.span_ops.resource@millisecond",
+    "spans.browser": "d:transactions/breakdowns.span_ops.ops.browser@millisecond",
+    "spans.db": "d:transactions/breakdowns.span_ops.ops.db@millisecond",
+    "spans.http": "d:transactions/breakdowns.span_ops.ops.http@millisecond",
+    "spans.resource": "d:transactions/breakdowns.span_ops.ops.resource@millisecond",
```

Comment:
Sounds good, although I would not know how to enforce that at the moment.

Reply:
yeah i think it's a good idea too, we can adjust this regular expression https://github.com/getsentry/sentry/blob/be6c69a31f2343a870b5e58a5c0cf253f2bb52f4/src/sentry/snuba/metrics/naming_layer/mri.py#L34-L34.

Reply:
In principle, the `<name>` part of the MRI is free-form, so what would a regex to check the name look like? I think the best way to prevent wrong MRIs is to define them only once and then use that constant / enum everywhere. In fact, an enum like that already exists: https://github.com/getsentry/sentry/blob/3c5d227638be1456feac761364c8422c33a51122/src/sentry/snuba/metrics/naming_layer/mri.py#L70

Reply:
yeah but sometimes we want to convert from string into Enum, that's why we have this RegExp.